build/.gitignore

```diff
@@ -1 +1,6 @@
-logs/
```

Comment:
logs was added for release.sh - please add back

Reply:
done